lib/bootstrap_form/helpers/bootstrap.rb

```diff
@@ -34,9 +34,11 @@ def alert_message(title, options = {})
       end
 
       def error_summary
-        content_tag :ul, class: 'rails-bootstrap-forms-error-summary' do
-          object.errors.full_messages.each do |error|
-            concat content_tag(:li, error)
+        if object.errors.full_messages.any?
```

Comment:
Could this be simplified to `object.errors.any?`, following the example of the Rails scaffold? https://github.com/rails/rails/blob/v5.2.0/railties/lib/rails/generators/erb/scaffold/templates/_form.html.erb.tt#L2